packages/meditrak-server/src/routes/postChanges.js

```diff
@@ -110,15 +117,15 @@ const constructEntitiesCreatedValidators = models => ({
 const constructIsValidEntity = models => async value =>
   new ObjectValidator(constructEntitiesCreatedValidators(models)).validate(value);
 
-const constructOptionsCreatedValidators = () => ({
+const constructOptionsCreatedValidators = models => ({
   id: [hasContent, takesIdForm],
   value: [hasContent],
-  option_set_id: [hasContent, takesIdForm],
+  option_set_id: [hasContent, takesIdForm, optionSetExist(models)],
```

Comment:
```suggestion
  option_set_id: [hasContent, takesIdForm, constructRecordExistsWithId(models.optionSet)],
```
And can delete your custom validator